packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -17,56 +17,71 @@ export const defaultServices = {
     return Auth.currentAuthenticatedUser();
   },
 
-  async handleSignUp(formData): Promise<any> {
-    return Auth.signUp(formData);
+  async handleSignUp({ username, password, attributes }, signUp): Promise<any> {
+    return signUp(username, password, attributes);
```

Comment:
Let's rethink about this API change. I have few concerns:
- parameters and return types are loosely typed. 
- parameters are manually typed instead of inferring from `Auth` library, and thus is hard to scale
- having a secondary `signUp` seems superflous, and I do think using `Auth.signUp` directly is a more intuitive DX (I know the current implementation comes from my idea, so apologies I'm backstabbing this)

I think we should try to avoid these API changes if we can make auto sign in change without it.

Reply:
Resolving, had an offline discussion to use documentation instead!